build/_k-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_DNXCORE='${E("KOREBUILD_TEST_DNXCORE")}'
+default CORECLR_RUNTIME_VERSION='${E("CORECLR_RUNTIME_VERSION")}'
```

Comment:
Why not update `$Path` for the DNX Core testing and have this file support executing only DNX Core tests the same way it executes only Mono tests on Linux? That is update `$Path` before calling `DnxTest()` or running `k-test` and have that macro / command use `dnx test` if the project targets `dnxcore50`.